src/task.rs

```diff
@@ -1098,6 +1098,12 @@ pub struct TaskMetrics {
     /// ```
     pub total_fast_poll_count: u64,
 
+    /// The total number of times that a task was executed swiftly.
+    ///
+    /// Here, 'swiftly' is defined as executing in strictly less time than
+    /// [`long_delay_threshold`][TaskMonitor::long_delay_threshold].
```

Comment:
Can you follow the documentation pattern of [`total_fast_poll_count`](https://docs.rs/tokio-metrics/0.2.0/tokio_metrics/struct.TaskMetrics.html#structfield.total_fast_poll_count) here?